regression/cbmc/function-return-no-body1/test.desc

```diff
@@ -5,7 +5,7 @@ activate-multi-line-match
 ^EXIT=10$
 ^SIGNAL=0$
 VERIFICATION FAILED
-<function_call hidden="false" step_nr="18" thread="0">\n\s*<function display_name="no_body" identifier="no_body">
-<function_return hidden="false" step_nr="19" thread="0">\n\s*<function display_name="no_body" identifier="no_body">
+<function_call hidden="false" step_nr="19" thread="0">\n\s*<function display_name="no_body" identifier="no_body">
+<function_return hidden="false" step_nr="20" thread="0">\n\s*<function display_name="no_body" identifier="no_body">
 --
```

Comment:
Is the step_nr really important? Perhaps make .*?

Reply:
Indeed, now using `\d+`.